### PR TITLE
Fix changelog line endings for git auto commit

### PR DESCRIPTION
**Pull request by mfcorgi**

### CODE REVIEW

**Comment by mfcorgi** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by mfcorgi** [comment]
recheck

**Comment by mairadanielaferrari** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by mairadanielaferrari** [comment]
recheck

**Comment by mfcorgi** [comment]
recheck

**Review by mrbiggred — APPROVED** [review]
Thank you for fixing the end of line issue.  I think the CLA check is failing for two reasons:

1) It still points to the "freshli" repo instead of "freshli-lib".
2) You appear to be using two separate GitHub accounts for this PR.  My guess is you have one account setup in your Git Client and the 2nd one when you login to the GitHub website.  It's probably best if you stuck with one GitHub account for all your actions.

The short version is I think the CLA check can be ignored for this PR.

File: .github/workflows/release-notes.yml [hunk 1]
```diff
@@ -67,7 +67,8 @@ jobs:
       # is not included in the GitReleaseManager action yet.
       - name: Generate Change Log
         run: |
-          dotnet-gitreleasemanager export --token ${{ secrets.PERSONAL_ACCESS_TOKEN }} -o 'corgibytes' -r 'freshli-lib' -f 'CHANGELOG.md'
+          dotnet-gitreleasemanager export --token ${{ secrets.GITHUB_TOKEN }} -o 'corgibytes' -r 'freshli-lib' -f 'CHANGELOG.md'
+          git add --renormalize CHANGELOG.md
```
[the remark this answers is not in the document]
Comment (mfcorgi) [reply .github/workflows/release-notes.yml:71]: > Thank you for fixing the end of line issue. I think the CLA check is failing for two reasons:
> 
> 1. It still points to the "freshli" repo instead of "freshli-lib".
> 2. You appear to be using two separate GitHub accounts for this PR.  My guess is you have one account setup in your Git Client and the 2nd one when you login to the GitHub website.  It's probably best if you stuck with one GitHub account for all your actions.
> 
> The short version is I think the CLA check can be ignored for this PR.

Yes, I noticed that I am using different git accounts. The corgi one and my personal one. Will fix that. hehe.
